src/gt4py/backend/dawn_backends.py

```diff
@@ -15,27 +15,42 @@
 # SPDX-License-Identifier: GPL-3.0-or-later
 
 import abc
+import enum
 import inspect
 import numbers
 import os
 import re
 import types
+import copy
```

Comment:
please sort the import alphabetically and add `from typing ...` from below as the last line of this block, since `typing` is part of the standard library